### DISCUSSION TITLE
Otel Collector Contrib pipeline exporters are not being build when Logs Exporters config contains azuremonitor

### DISCUSSION THREAD

**Discussion by romm333** [open-telemetry/opentelemetry-collector-contrib#10880] [Q&A]
Hi, I'm just trying to double check before starting an issue - is it something wrong with my setup or it is a bug in the  azuremonitor exporter? When logs section of the  collector config contains "azuremonitor" exporter, otel collector does not start in a docker-compose with message  **Error: cannot build exporters: pipeline "logs" of data type "logs" has an exporter azuremonitor, which does not support that data type"** 

pipeline setup:

service:
  pipelines:
    traces:
      receivers: [otlp]
      processors: [batch]
      exporters: [logging, zipkin, azuremonitor]
    metrics:
      receivers: [otlp]
      processors: [batch]
      exporters: [logging, prometheus]
    logs:
      receivers: [otlp]
      processors: []
      exporters: [logging, **azuremonitor**]

Has anybody come across the same  problem?

Br

**Comment by jpkrohling** [comment]
cc @pcwiese as the code owner
